lms/templates/instructor/edx_ace/allowedenroll/email/body.html

```diff
@@ -42,18 +52,26 @@ <h1>
 
             <p style="color: rgba(0,0,0,.75);">
                 {% if auto_enroll %}
+                {% filter force_escape %}
```

Comment:
Can you align the `filter` with the `blocktrans` here and below so the if/elif/else stand out?

Reply:
Fixed